src/index.html

```diff
@@ -1,12 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="body">
+    <div class="calendar calendar--start-day-sun calendar--month-length-29">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--29">
```
Better use day -- sun, because sun it's the value of the modifier.